src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -95,13 +95,20 @@
  * </li>
  * </ul>
  * <p>
- * Example to allow multiple annotations on the same line
+ * Default configuration:
  * </p>
  * <pre>
- * &#64;SuppressWarnings("deprecation") &#64;Mock DataLoader loader; // no violations
+ * &lt;module name=&quot;AnnotationLocation&quot;/&gt;
  * </pre>
  * <p>
- * Use the following configuration:
+ * Example for default configuration:
+ * </p>
+ * <pre>
+ * &#64;Override public int hashCode() { ... } // ok
```

Comment:
please change `...` to `return 1;` lets keep code compilable. Same for others.

please add examples (to show that we are ok with annotations on previous line by default, example of completely different treatment of google style - https://github.com/google/google-java-format/issues/360 ):
```
@NotNull private boolean field1;
@NotNull
private boolean field2;
@Override
public boolean equals(Object obj) { }
```

Reply:
Done